plotly/tests/test_optional/test_jupyter/test_jupyter.py

```diff
@@ -12,10 +12,17 @@
 import subprocess
 
 PATH_ROOT = path.dirname(__file__)
+PATH_NODE_MODULES = path.join(PATH_ROOT, 'node_modules')
 PATH_FIXTURES = path.join(PATH_ROOT, 'fixtures')
 PATH_JS_TESTS = path.join(PATH_ROOT, 'js_tests')
 
 
+class PlotlyJupyterTestDeps(TestCase):
+
+    def test_node_modules(self):
+        self.assertTrue(path.isdir(PATH_NODE_MODULES))
```

Comment:
@theengineear this test will fail when someone forgets to `npm install` before running these tests. It should make debugging a little easier.